Foreword/Foreword_DV_transliterated.txt

```diff
@@ -1,112 +1,111 @@
+ކީރިތި ޤުރްއާން ދިވެހި ބަހަށް ތަރުޖަމާ ކުރުމުގެ މަސައްކަތް ފެށުނީއްސުރެ، އެކަމަށް އެހީތެރިވެ ދެއްވުމުގެ ގޮތުން، ލީބިއާގެ ޖަމްޢިއްޔަތުއް ދަޢުވަތުލް އިސްލާމިއްޔަތުލް ޢާލަމިއްޔާގެ ފަރާތުން ވެދެއްވި ދީލަތި އެހީގެ އަގު ވަޒަންކޮށް، އެ ޖަމްޢިއްޔާގެ ބާނީ އަލްޢަޤީދު މުޢައްމަރު އަލްޤައްޛާފީ އަށާއި، އެ ޖަމްޢިއްޔާގެ މަސްއޫލު ވެރިންނަށާއި، ލީބިއާގެ ސަރުކާރަށް، ނުހަނު އިޙްލާޞްތެރިކަމާއެކު ޝުކުރު ދަންނަވަމެވެ.
 
-ކީރިތި ޤުރުއާނުގެ މި ތަރުޖަމާގައިހުރި، އެއްކަހަލަ އާޔަތްތަކުގެ މާނަ އެއްގޮތް ކުރެއްވުމަށްޓަކައިވެސް، ޙާއްޞަ ކޮމިޓިއެއް ހޮވައި، އެކޮމެޓީއާއި އެ މަސައްކަތް އަޅުގަނޑު ހަވާލުކުރީމެވެ. އެ ކޮމިޓީގައި ހިމެނިވަޑައިގެންނެވީ:
+ކީރިތި ޤުރްއާނުގެ މި ތަރުޖަމާގައިހުރި، އެއްކަހަލަ އާޔަތްތަކުގެ މާނަ އެއްގޮތް ކުރެއްވުމަށްޓަކައިވެސް، ޙާއްޞަ ކޮމިޓިއެއް ހޮވައި، އެކޮމެޓީއާއި އެ މަސައްކަތް އަޅުގަނޑު ހަވާލުކުރީމެވެ. އެ ކޮމިޓީގައި ހިމެނިވަޑައިގެންނެވީ:
 
-1. ފަޟީލަތުއް ޝައިޚު މުޙައްމަދު ޤުބާދު އަބޫބަކުރު (ފިޔާތޮށިގެ/ޅ.ނައިފަރު )
+1. ފަޟީލަތުއް ޝައިޚު މުޙައްމަދު ޤުބާދު އަބޫބަކުރު (ފިޔާތޮށިގެ/ޅ.ނައިފަރު)
 
 2. ފަޟީލަތުލް އުސްތާޛު އާދަމް މުޙައްމަދު އަބްދުﷲ (ފަށުވިމާގެ/ލ.މާވަށް)
 
 3. ފަޟީލަތުލް އުސްތާޛު ޙަސަން ސައީދް (ރީތިގަސްދޮސުގެ/ސ.ހިތަދޫ)
 
 މިބޭބޭފުޅުންނެވެ.
 
-އަދި އޭގެފަހުން، ކީރިތި ޤުރުއާނުގެ ދިވެހި ތަރުޖަމާ ޗާޕްކުރުމަށް ތައްޔާރުކުރުމުގެ ގޮތުން، ކޮންޕިއުޓަރުން ލޭއައުޓް ހެއްދެވުމަށް، އެފަދަ މަސައްކަތްތައް ދެނެވަޑައިގަންނަވާ ހުނަރުވެރި ބޭފުޅުންކޮޅަކާ އެ މަސައްކަތް ހަވާލު ކުރީމެވެ. އެބޭފުޅުންނަކީ:
+އަދި އޭގެފަހުން، ކީރިތި ޤުރްއާނުގެ ދިވެހި ތަރުޖަމާ ޗާޕްކުރުމަށް ތައްޔާރުކުރުމުގެ ގޮތުން، ކޮންޕިއުޓަރުން ލޭއައުޓް ހެއްދެވުމަށް، އެފަދަ މަސައްކަތްތައް ދެނެވަޑައިގަންނަވާ ހުނަރުވެރި ބޭފުޅުންކޮޅަކާ އެ މަސައްކަތް ހަވާލު ކުރީމެވެ. އެބޭފުޅުންނަކީ:
 
 1. ފަޟީލަތުލް އުސްތާޛު ޙަސަން ސަޢީދު (ރީތިގަސްދޮށުގެ/ސ.ހިތަދޫ)
 
 2. އަލްފާޟިލް އަޙްމަދު ރަޝީދު އަބްދުލްޤާދިރު (މ.މަގުމަތީގެ)
 
 3. ފަޟީލަތުލް އުސްތާޛު މޫސާ އަންވަރު ޙަސަން (ރިހިވިލާގެ/ޏ.ފުވައްމުލައް)
 
-4. ފަޟީލަތުލްއުސްތާޛު އަބްދުﷲ ޙަމީދު (ނޫރާނީމާގެ/ނ.ހެނބަދޫ)
+4. ފަޟީލަތުލް އުސްތާޛު އަބްދުﷲ ޙަމީދު (ނޫރާނީމާގެ/ނ.ހެނބަދޫ)
 
 5. ފަޟީލަތުލް އުސްތާޛު ޙާމިދު އަބޫބަކުރު (ފިޔާތޮށިގެ/ޅ.ނައިފަރު)
 
 މިބޭބޭފުޅުންނެވެ.
 
-މި ބޭފުޅުން އެ މަސައްކަތްތައް ނިންމެވުމުން، ދިވެހި ބަހުގެ ރިވެތި ބަސްމަގުގެގޮތުން ތަރުޖަމާ ފުރިހަމަ ކުރުމަށްޓަކައި، ރާއްޖޭގައި މިހާރު ހުންނެވި ދިވެހި ބަހުގެ ވަރަށްމޮޅު އަދީބެއް ކަމުގައިވާ ހ.އާމުލީގޭ އަލްފާޟިލް އަބްދުﷲ ޞާދިޤާ އެ މަސައްކަތް ހަވާލުކުރީމެވެ. އޭނާ އެ މަސައްކަތް ވަރަށް ފަރުވާތެރިކަމާއެކު ކޮށްދެއްވިއެވެ. އެ ކަމަށްޓަކައި އަލްފާޟިލް އަބްދުﷲ ޞާދިޤަށް ވަރަށް އިޙްލާޞްތެރިކަމާއެކު ޝުކުރު ދަންނަވަމެވެ.
+މި ބޭފުޅުން އެ މަސައްކަތްތައް ނިންމެވުމުން، ދިވެހި ބަހުގެ ރިވެތި ބަސްމަގުގެގޮތުން ތަރުޖަމާ ފުރިހަމަ ކުރުމަށްޓަކައި، ރާއްޖޭގައި މިހާރު ހުންނެވި ދިވެހި ބަހުގެ ވަރަށްމޮޅު އަދީބެއް ކަމުގައިވާ ހ.އާމުލީގޭ އަލްފާޟިލް އަބްދުﷲ ޞާދިޤުއާ އެ މަސައްކަތް ހަވާލުކުރީމެވެ. އޭނާ އެ މަސައްކަތް ވަރަށް ފަރުވާތެރިކަމާއެކު ކޮށްދެއްވިއެވެ. އެ ކަމަށްޓަކައި އަލްފާޟިލް އަބްދުﷲ ޞާދިޤަށް ވަރަށް އިޙްލާޞްތެރިކަމާއެކު ޝުކުރު ދަންނަވަމެވެ.
```

Comment:
Technically the original version is the correct one (but with ޢަބްދުﷲ instead އަބްދުﷲ)

ޢަބްދުﷲ ޞާދިޤާ

There's also other instances where it's used that way

އިބްރާހީމް އިސްމާޢީލާއި

Reply:
will fix those